spec/controllers/questionnaires_controller_spec.rb

```diff
@@ -1,4 +1,20 @@
-describe QuestionnairesController do
+extend RSpec::Matchers
+require 'spec_helper'
+
+RSpec.configure do |config|
+  config.expect_with :rspec do |c|
+    # Disable the `expect` sytax...
+    c.syntax = :should
+
+    # ...or disable the `should` syntax...
+    c.syntax = :expect
+
+    # ...or explicitly enable both
+    c.syntax = [:should, :expect]
+  end
+end
+
+describe QuestionnairesController, type: :controller do
```

Comment:
You do not need to add `type: :controller` here. RSpec will figure it out automatically.